keyboards/handwired/ep40/info.json

```diff
@@ -0,0 +1,13 @@
+{
+    "keyboard_name":"ep40",
+    "url":null,
+    "maintainer":"e11i0t23",
+    "width":12,
+    "hight":4,
+    "layouts": {
+        "LAYOUT":{
+            "key_count":43,
+            "layout":[]
```

Comment:
This is an incomplete info.json. You can make this by making a KLE of what your layout looks like, and then copy pasting the raw into https://qmk.fm/converter/